rules/0131_test.go

```diff
@@ -0,0 +1,223 @@
+// Copyright 2019 Google LLC
+//
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package rules
+
+import (
+	"testing"
+
+	"github.com/googleapis/api-linter/rules/testutil"
+)
+
+func TestRequestMessageName(t *testing.T) {
+	tmpl := testutil.MustCreateTemplate(`
+	syntax = "proto3";
+
+	service Aip131 {
+		rpc GetFoo({{ .RequestName }}) returns (Foo);
+	}
+
+	message {{.RequestName }} {
+		string name = 1;
+	}
+
+	message Foo {}
+	`)
+
+	tests := []struct {
+		RequestName  string
+		problemCount int
+		suggestion   string
+		startLine    int
+	}{
+		{"GetFooRequest", 0, "", -1},
+		{"GetFooReq", 1, "GetFooRequest", 5},
+	}
+
+	rule := checkRequestMessageName()
+
+	for _, test := range tests {
+		req := testutil.MustCreateRequestFromTemplate(tmpl, test)
```

Comment:
Consider using t.Run to run each test.

Reply:
I blindly copied this section. I defer to you and @mingzhi.